tests/Feature/CreateUserFeatureTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Hash;
+use Tests\TestCase;
+
+class CreateUserFeatureTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function testCreateUserPageLoads()
+    {
+        $response = $this->get('/users/create');
+        $response->assertStatus(200);
+    }
+
+    public function testFormCreatesUserName()
+    {
+        $this->assertDatabaseCount('users', 0);
+
+        $this->post('users', [
+            'name' => 'John Doe',
+            'email' => 'email@test.ca',
+            'password' => 'password123',
+            'password_confirmation' => 'password123'
```

Comment:
again, this should rely on a user factory instead

Reply:
here it makes sense to use a factory